Packs/CortexXDR/Integrations/CortexXDRIR/README.md

```diff
@@ -2508,3 +2508,57 @@ Initiates a new endpoint script execution kill process.
 >|action_id|endpoints_count|
 >|---|---|
 >| 3658 | 1 |
+
+
+
+### xdr-endpoint-scan-abort
+***
+Cancel the scan of selected endpoints. A scan can only be aborted if the selected endpoints are in Pending or in Progress status. To scan all endpoints, run this command with argument all=true. Do note that scanning all the endpoints may cause performance issues and latency.
+
+#### Base Command
+
+`xdr-endpoint-scan-abort`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| endpoint_id_list | List of endpoint IDs. | Optional | 
+| dist_name | Name of the distribution list. | Optional | 
+| gte_first_seen | Integer in timestamp epoch milliseconds. | Optional | 
+| gte_last_seen | Integer in timestamp epoch milliseconds. | Optional | 
+| lte_first_seen | Integer in timestamp epoch milliseconds. | Optional | 
+| lte_last_seen | Integer in timestamp epoch milliseconds. | Optional | 
+| ip_list | List of IP addresses. | Optional | 
+| group_name | Name of the endpoint group. | Optional | 
+| platform | Type of operating system. Possible values are: windows, linux, macos, android. | Optional | 
+| alias | Endpoint alias name. | Optional | 
+| isolate | Whether an endpoint has been isolated. Can be "isolated" or "unisolated". Possible values are: isolated, unisolated. | Optional | 
+| hostname | Name of the host. | Optional | 
+| all | Whether to scan all of the endpoints or not. Default is false. Do note that scanning all the endpoints may cause performance issues and latency. Possible values are: true, false. Default is false. | Optional | 
+
+
+#### Context Output
+
+There is no context output for this command.
```

Comment:
do not forget to add the scanID and the aborted keys.

Reply:
done